composer.lock

```diff
@@ -235,7 +175,69 @@
                     "type": "community_bridge"
                 }
             ],
-            "time": "2020-11-19T20:18:59+00:00"
+            "time": "2021-09-02T16:11:32+00:00"
+        },
+        {
+            "name": "laminas/laminas-zendframework-bridge",
```

Comment:
This dependency sneaked into the `require-dev` section: can we somehow get rid of it? I think an aggressive bump of `laminas/laminas-stdlib` should suffice for that.

Reply:
It's actually installed due to the laminas-coding-standard requirement (this repo is still on v1 of the standard):

```bash
$ composer why laminas/laminas-zendframework-bridge   
laminas/laminas-coding-standard  1.0.0  requires  laminas/laminas-zendframework-bridge (^1.0)
```

Reply:
Can we get rid of it? I'd be more comfortable with the tests being green even without `laminas/laminas-zendframework-bridge` :thinkin

Reply:
Updated to laminas-coding-standard 2.2 series which eliminates this requirement, and also removes the file-level docblocks.